k2/csrc/math.h

```diff
@@ -24,9 +26,55 @@ namespace k2 {
   HighestBitSet(1) = 0,
   HighestBitSet(2,3) = 1
   ...
+
+  Note we may delete this function later if there's no other usage.
  */
 int32_t HighestBitSet(int32_t i);
 
+// Generate a uniformly distributed random variable of type int32_t.
+class RandIntGenerator {
```

Comment:
This class is copied from https://github.com/k2-fsa/k2/blob/072853691083239fa93220de1956bc642fb2a8c5/k2/csrc/host/fsa_util.cc#L32-L49, we may need to delete that function in `fsa_util.cc` in the future.